File: LocalHiringPlatform.Domain/Models/NotificationModel.cs

namespace LocalHiringPlatform.Domain.Models
{
    public class NotificationModel
    {
        public Guid EntityId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
        public bool IsRead { get; set; }
        public DateTime CreatedOn { get; set; }
    }
}

File: UI/localhiringplatform-ui/src/types/SignalR/NotificationModel.ts
import type { NotificationTypes } from "../../Enums/NotificationType";

export interface NotificationModel {
    entityId: string;
    type: NotificationTypes; // Assuming NotificationType is an enum or union type defined elsewhere
    title: string;
    message: string;
    isRead: boolean;
    createdOn: Date | string; // Use Date if parsed, or string if raw ISO timestamp from JSON
}